containers/php/fpm/php-fpm-healthcheck

```diff
@@ -0,0 +1,139 @@
+#!/bin/sh
```

Comment:
Wie bekommen wir mit, dass es zu diesem Script Updates gibt und wie sieht der Updateprozess aus?

Reply:
The update process is manual. The last release is from 2020 and the last commit happened 10 months ago. The source of the script is mentioned in the comments. I think we don't have to worry about the update.